core/modification/src/main/java/datawave/modification/MutableMetadataUUIDHandler.java

```diff
@@ -322,13 +329,7 @@ else if (f.getName().equalsIgnoreCase(event.getIdType()) && fieldCount < 1 && co
                                         log.trace("Submitting request to MutableMetadataHandler from MutableMetadataUUIDHandler: " + modReq);
 
                                     // make sure user isn't null or empty
```

Comment:
this is no longer being checked add the null/empty check or remove the comment

Reply:
on it.